pkg/authenticator/version.go

```diff
@@ -4,7 +4,7 @@ import "fmt"
 
 // Version field is a SemVer that should indicate the baked-in version
 // of the authn-k8s-client
-var Version = "0.19.1"
+var Version = "0.19.2"
```

Comment:
i think that the `RH base image` change (which does not have a linked issue btw) is a change that requires a MINOR version bump so we should change the version to `0.20.0`).

wdyt?

Reply:
Works for me. I also added the missing PR link.